CHANGELOG/CHANGELOG-1.21.md

```diff
@@ -413,4 +413,4 @@ _Nothing has changed._
 
 ### Removed
 - rsc.io/quote/v3: v3.1.0
-- rsc.io/sampler: v1.3.0
\ No newline at end of file
+- rsc.io/sampler: v1.3.0
```

Comment:
can we remove this delta as well? 1.21 file should not be here at all

Reply:
updated.